### PR TITLE
scripts: Add support to build ppc64le binary for release

### PR DESCRIPTION
**Pull request by mkumatag**
This PR is for building ppc64le binary for release.

[contributing]: https://github.com/coreos/etcd/blob/master/CONTRIBUTING.md#contribution-flow

### CODE REVIEW

File: scripts/build-binary [hunk 1]
```diff
@@ -59,23 +59,33 @@ function main {
...
-		mkdir ${TARGET}
-		package ${TARGET} ${PROJ}
-
-		if [ ${GOOS} == "linux" ]; then
-			tar cfz ${TARGET}.tar.gz ${TARGET}
-			echo "Wrote release/${TARGET}.tar.gz"
-		else
-			zip -qr ${TARGET}.zip ${TARGET}
-			echo "Wrote release/${TARGET}.zip"
+		if [[ $GOOS == "linux" ]]; then
```
Comment (heyitsanthony) [inline scripts/build-binary]:
```bash
if [ ${GOOS} == "linux" ]; then
```
to match the rest of the file's style

File: scripts/build-binary [hunk 2]
```diff
@@ -59,23 +59,33 @@ function main {
...
+			TARGET_ARCHS+=("ppc64le")
 		fi
+
+		for ARCH in "${TARGET_ARCHS[@]}"
```
Comment (heyitsanthony) [inline scripts/build-binary]:
```bash
for TARGET_ARCH in "${TARGET_ARCHS[@]}"; do
```
to match file style

Comment (mkumatag) [inline scripts/build-binary]: addressed in latest commit, PTAL

File: scripts/build-binary (hunk 1 above)
Comment (mkumatag) [inline scripts/build-binary]: addressed in the latest commit, PTAL

**Comment by mkumatag** [comment]
@heyitsanthony I hope this commit is good enough to see ppc64le binaries in the release page!

**Comment by heyitsanthony** [comment]
@mkumatag sure, but will probably have to wait until 3.2-alpha since 3.1 backports are for bugfixes...

File: scripts/build-binary (hunk 1 above)
Comment (heyitsanthony) [inline scripts/build-binary]: update didn't take? not seeing the changes...

**Comment by mkumatag** [comment]
@heyitsanthony sure, thanks.

File: scripts/build-binary (hunk 1 above)
Comment (mkumatag) [inline scripts/build-binary]: I do see changes! https://github.com/coreos/etcd/pull/7262/files#diff-a8e6fe6de9d13f0daabafff276e9a583R64

File: scripts/build-binary [hunk 3]
```diff
@@ -59,23 +59,33 @@ function main {
...
-		mkdir ${TARGET}
-		package ${TARGET} ${PROJ}
-
-		if [ ${GOOS} == "linux" ]; then
-			tar cfz ${TARGET}.tar.gz ${TARGET}
-			echo "Wrote release/${TARGET}.tar.gz"
-		else
-			zip -qr ${TARGET}.zip ${TARGET}
-			echo "Wrote release/${TARGET}.zip"
+		if [[ ${GOOS} == "linux" ]]; then
```
Comment (heyitsanthony) [inline scripts/build-binary]: this is still `if [[ ${GOOS} == "linux" ]]; then` it should be `if [ ${GOOS} == "linux"]; then` to match the style below

File: scripts/build-binary [hunk 4]
```diff
@@ -59,23 +59,33 @@ function main {
...
+			TARGET_ARCHS+=("ppc64le")
 		fi
+
+		for TARGET_ARCH in "${TARGET_ARCHS[@]}"
```
Comment (heyitsanthony) [inline scripts/build-binary]: this should be `for TARGET_ARCH in "${TARGET_ARCHS[@]}"; do` to match the for loop style above

**Comment by heyitsanthony** [comment]
lgtm. Thanks!

File: scripts/build-binary (hunk 3 above)
Comment (mkumatag) [inline scripts/build-binary]: my bad! fixed in latest commit. PTAL

File: scripts/build-binary (hunk 4 above)
Comment (mkumatag) [inline scripts/build-binary]: my bad! fixed in latest commit. PTAL

**Comment by mkumatag** [comment]
ping @gyuho

**Comment by gyuho** [comment]
Thanks!

**Pull request merged**